### DISCUSSION TITLE
Archive a complete wordpress site?

### DISCUSSION THREAD

**Discussion by spiffzilla** [ArchiveBox/ArchiveBox#671] [Q&A] [answered]
I'm trying to archive https://knaften.se with posts and all, but all I get is the landing page. No posts are archived, is there a way to archive the whole site?

**Comment by pirate** [comment, marked as answer]
See https://github.com/ArchiveBox/ArchiveBox/issues/191

**Reply by spiffzilla** [reply]
Thank you, my cheeks are red with the could-not-search-for-the-answer shame.

**Reply by pirate** [reply]
haha no worries
